orchestrator-salt/src/main/resources/salt-common/salt/fluent/settings.sls

```diff
@@ -140,6 +140,8 @@
   {% set proxy_full_url = None %}
 {% endif %}
 
+{% set forward_port = 24224 %}
```

Comment:
Can this port change?

Reply:
that's typically fluentd port, only needs to be changed if some reason someone setup a fluentd with recipes and using that port